common/migrations/0001_create_superuser.py

```diff
@@ -0,0 +1,29 @@
+"""Module for creating DB-admin."""
+from django.db import migrations
+from django.contrib.auth.hashers import make_password
+import environ
+
+env = environ.Env(DEBUG=(bool, False),)
```

Comment:
1) Please don't forget reading `.env` into environment.
2) `DEBUG` var is not needed here.
3) It is better to move declaration of `ADMIN_EMAIL` into settings and just import it from there. Keeping all related things in one place helps a lot :)